superset/models/sql_lab.py

```diff
@@ -211,6 +211,19 @@ class SavedQuery(Model, AuditMixinNullable, ExtraJSONMixin, ImportExportMixin):
     def __repr__(self) -> str:
         return str(self.label)
 
+    def to_dict(self) -> Dict[str, Any]:
+        return {
+            "id": self.id,
+            "schema": self.schema,
+            "label": self.label,
+            "description": self.description,
+            "sql": self.sql,
+            "userId": self.user_id,
```

Comment:
I wonder if someone were to share a saved query, would this return the original owner's user id?

Reply:
That's a great question. The only thing that we are currently using in here is the id, so I could just limit it to that for now?

Reply:
yeah, that sounds like a safe solution.